### ISSUE TITLE
iBus [Keyman] Integration

### ISSUE THREAD

**Issue by Ilhamkang** [keshavbhatt/whatsie#15]
I would like to thanks to you for developing this app. This is so useful! One thing I found that any keyboard layout installed through iBus or Keyman did not work to type on Whatsie. Can you fix the problem? 

Regards, 
Ilham

**Comment by keshavbhatt** [comment]
Hi, this is a known issue https://forum.snapcraft.io/t/ibus-doesnt-work-with-skype-snap-on-20-04/17279
I have linked this issue with the forum post.

**Comment by Ilhamkang** [comment]
Aha, okay. Thank you! 
Hopefully there is a way to resolve it.

**Comment by keshavbhatt** [comment]
I have updated the content snap with a fix that may solve the issue, you can test the update by updating the qt513 content snap using `snap refresh qt513 --edge` command to confirm.

**Comment by Ilhamkang** [comment]
I have tried the update, but it seems not affected to the problem. 
I'm using Xubuntu (Ubuntu 20.04.2 LTS x86_64 )  Kernel: 5.4.0-80-generic, IBus 1.5.22, and Keyman 14.0 

Here is the recorded input I've made. 

https://user-images.githubusercontent.com/22653757/127280437-407c9787-c5be-468b-9e7a-28d26a7f46a7.mp4

**Comment by keshavbhatt** [comment]
I have tested with Ubuntu 16.04.7 with IBus 1.5.11 woking fine here:

https://user-images.githubusercontent.com/5965031/127281896-8b148802-616a-48be-bf68-3fa831a06a59.mp4

**Comment by keshavbhatt** [comment]
**Also tested on** 
```
Distributor ID:	Ubuntu
Description:	Ubuntu 21.04
Release:	21.04
Codename:	hirsute
```

Which is using xkb for keyboard input, the output of `apt-cache policy xkb-data
` is:
```
xkb-data:
  Installed: 2.29-2build1
  Candidate: 2.29-2build1
  Version table:
 *** 2.29-2build1 500
        500 http://archive.ubuntu.com/ubuntu hirsute/main amd64 Packages
        500 http://archive.ubuntu.com/ubuntu hirsute/main i386 Packages
        100 /var/lib/dpkg/status
```
Everything works fine.

**Comment by Ilhamkang** [comment]
I'm afraid the problem that I found is specific to my machine only, or perhaps there is a problem with Ubuntu 20.04 core version? I'm not really sure. Upgrading to 21.04 is considerable, but at the mean time I will stick to the version I'm using now. The input problem is not a major for me, at least, since I still can reproduce the non-Latin character through copy-paste from the text editor.      

Thank you anyway for trying to fix this. I will let you know if there is another condition which makes it work or not to my machine. 

Your best!

**Comment by keshavbhatt** [comment]
The issue seems to be fixed in 0483b030043f13fe048f575882ffb78d4177b479
You can refresh whatsie with `snap refresh whatsie --stable` to get the changes.
Feel free to reopen for further assistance.

**Comment by shakaran** [comment]
@Ilhamkang, for iBus and Keyman input, the maintained fork **Whatly** runs on current Qt6 WebEngine, where input-method integration is much improved over the old Qt5 build. Worth retrying. https://github.com/shakaran/whatly
